README.md

```diff
@@ -26,5 +26,6 @@ Running `4` you should see roughly a 10% improvement versus `3`.
 To get started, generate a flame graph for the first server:
 
 ```sh
-clinic flame --on-port 'autocannon localhost:$PORT' -- node 1-server-with-slow-function.js
+clinic flame --autocannon [ 'localhost:$PORT' ] -- node
+1-server-with-slow-function.js
```

Comment:
`--autocannon` handles the localhost:$PORT stuff internally—you only have to specify the URL path. In this case, `/`.

(also this should be one line :D )

Reply:
Ah I'm used to the 80 character line limit from working on node

Reply:
That's normally a fine guideline! But inside a command-line code sample it changes the meaning, because the newline will end the command.